ktor-server/ktor-server-cio/common/src/io/ktor/server/cio/HttpServer.kt

```diff
@@ -32,9 +32,32 @@ public class HttpServer(
  * @property connectionIdleTimeoutSeconds time to live for IDLE connections
  * @property reuseAddress allow the server to bind to an address that is already in use
  */
-public data class HttpServerSettings(
-    val host: String = "0.0.0.0",
-    val port: Int = 8080,
-    val connectionIdleTimeoutSeconds: Long = 45,
-    val reuseAddress: Boolean = false
+public open class HttpServerSettings(
```

Comment:
Changing this class from `data class` to plain `class` is a backwards incompatible change. Probably we should introduce a new interface (without `host` and `port`) for `HttpServerSettings` and use it for `UnixSocketServerSettings`

Reply:
let me check if it is possible